### PR TITLE
Go global

### PR DESCRIPTION
**Pull request by laurieboyes**
Set the schedules for asia and americas

### CODE REVIEW

File: functions/get-users/test/get-users.spec.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const users = Array(4).fill().map(() => uuid());
 const timezones = [
 	'America/New_York',
 	'Europe/London',
-	'Asia/Tokyo',
+	'Asia/Hong_Kong',
 	'Europe/London'
```
Comment (debugwand) [inline functions/get-users/test/get-users.spec.js]: duplicate

**Comment by debugwand** [comment]
switches to summer time saving are not universally on the same day

**Comment by laurieboyes** [comment]
Hm, yes this timezone stuff is pretty grim. I think I'm happy to accept this level of inaccuracy, as we've already accepted it to some degree by only offering 3 choices. As long as it comes at _roughly_ 5/6 on each time, I think that's ok, and I don't think we should spend any time working on any more complex logic.

**Comment by keirog** [comment]
Afaik, times in the config should be set to the time _in that timezone_, so all daily entries should basically have the same value (5) in the config object.

**Pull request merged**